_data/navigation.yml

```diff
@@ -63,6 +63,10 @@ setup:
             url: /setup/install/providers/kubernetes/
           - title: Kubernetes (Manifest Based)
             url: /setup/install/providers/kubernetes-v2/
+            collapsed: false
+            children:
+              - title: Amazon EKS
+                url: /setup/install/providers/kubernetes-v2/aws-eks/
```

Comment:
I think you added to main content pages: one under /setup/providers/aws/ and this one, under /setup/providers/kubernetes-v2/.  But you added only one to this nav?